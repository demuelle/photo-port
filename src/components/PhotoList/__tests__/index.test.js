import { render, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import PhotoList from '..'

const portrait = { name: "portraits", description: "Portraits of people in my life" };

afterEach(cleanup)

describe('Gallery is rendering', () => {

  it('renders', () => {
    render(<PhotoList currentCategory={portrait} />);
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<PhotoList currentCategory={portrait} />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('renders', () => {
    const { getByTestId } = render(<PhotoList currentCategory={portrait} />)
    expect(getByTestId('photo-flex-row')).toBeTruthy();
  })
})