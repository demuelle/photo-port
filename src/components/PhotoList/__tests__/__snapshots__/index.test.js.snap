// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery is rendering matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="flex-row"
      data-testid="photo-flex-row"
    >
      <img
        alt="Cat green eyes"
        class="img-thumbnail mx-1"
      />
      <img
        alt="Green parrot"
        class="img-thumbnail mx-1"
      />
      <img
        alt="Yellow macaw"
        class="img-thumbnail mx-1"
      />
      <img
        alt="Pug smile"
        class="img-thumbnail mx-1"
      />
    </div>
  </div>
</DocumentFragment>
`;
